Ably.podspec

```diff
@@ -16,8 +16,8 @@ Pod::Spec.new do |s|
   s.osx.deployment_target   = '10.12'
   s.requires_arc            = true
   s.swift_version           = '5.0'
-  s.source_files            = 'Source/**/*.{h,m,swift}', 'SocketRocket/SocketRocket/**/*.{h,m}'
-  s.private_header_files    = 'Source/*+Private.h', 'SocketRocket/SocketRocket/internal/**/*.h'
+  s.source_files            = 'Source/**/*.{h,m,swift}'
+  s.private_header_files    = 'Source/*+Private.h', 'Source/SocketRocket/SocketRocket/**/*.h'
```

Comment:
Yeah, I did think about it but I was a little uneasy — it's handy having it in its own directory because it gives us an easy way to keep the `LICENCE` file alongside the code that it applies to. I also thought it might be useful for future developers to have some way of knowing that this code wasn't written by us. Perhaps it's something we could do in the future?

Reply:
Sorry, I may have misunderstood, actually. Do you mean getting rid of the `Source/SocketRocket` directory, or the `Source/SocketRocket/SocketRocket` directory? I thought you meant the first one, but if it's the second then yeah I agree we could do that.

Reply:
Yeah, I mean `Source/SocketRocket/SocketRocket` and move LICENSE into `Source/SocketRocket`.

Reply:
My reading of the `LICENSE` file is that we can't remove it – the `LICENSE` file says (emphasis mine):

>  * Redistributions in binary form must reproduce the above copyright notice,
   _this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution._

and the header comments only refer to the contents of `LICENSE` file; they don't reproduce it:

> //
> // This source code is licensed under the BSD-style license found in the
> // LICENSE file in the root directory of this source tree. An additional grant
> // of patent rights can be found in the PATENTS file in the same directory.
> //

I'll remove the intermediate directory, though.

Reply:
Done 👍